Makefile

```diff
@@ -98,6 +103,38 @@ zip: compile transcompile
 	find /tmp/$(PLUGINNAME) -iname "*.pyc" -delete
 	cd /tmp; zip -9r $(CURDIR)/$(PLUGINNAME).zip $(PLUGINNAME)
 
+installer: zip
+	#Prerequisite is to have a plugin zip ready in the ThreeDiToolbox repo folder.
```

Comment:
Comment no longer needed. Maybe add a warning that this command will download a lot of data and can take a long while.